lib/realtime/helpers.ex

```diff
@@ -313,9 +313,13 @@ defmodule Realtime.Helpers do
   """
   @spec kill_connections_to_tenant_id_in_all_nodes(String.t(), atom()) :: :ok
   def kill_connections_to_tenant_id_in_all_nodes(tenant_id, reason) do
-    Enum.each(Node.list(), fn node ->
-      :erpc.call(node, __MODULE__, :kill_connections_to_tenant_id, [tenant_id, reason], 5000)
-    end)
+    Task.async_stream(
+      Node.list(),
```

Comment:
I guess it will be invoked on all other nodes, but not on the current one, correct?

Reply:
you are right, I need to add node() to this list, great catch thank you